hack/jenkins/test-dockerized.sh

```diff
@@ -55,4 +55,3 @@ go install ./cmd/...
 
 make test-cmd
 make test-integration
-./hack/test-update-storage-objects.sh
```

Comment:
We should remove the script as well. Tracked in https://github.com/kubernetes/kubernetes/issues/69299.